getnacelle/nacelle-js
# fix: replace non-relay queries in Next

### WHY are these changes introduced?

Fixes [[ENG-7319](https://nacelle.atlassian.net/browse/ENG-7319)] 



### WHAT is this pull request doing?



## 

- replace non-relay queries with relay queries

### How to Test

- `npm run build && npm run start` (to test the route building with the new queries)
- make sure homepage works and all the content in it shows properly
- make sure the cart works
- navigate to a product page `product/handle`
- navigate to a collection page `collections/handle` -> use pagination
- navigate to a content page `content/handle`

### Note

I couldn't get prettier to format the GraphQL for me so I did my best to do it manually, if prettier formats it differently on your side feel free to contribute to this PR or to LMK!
